packages/@lwc/integration-not-karma/configs/hydration.js

```diff
@@ -14,4 +14,13 @@ export default {
     ...baseConfig,
     files: ['test-hydration/**/*.spec.js', '!test-hydration/synthetic-shadow/index.spec.js'],
     plugins: [...baseConfig.plugins, hydrationTestPlugin],
+    coverageConfig: {
+        ...baseConfig.coverageConfig,
+        threshold: {
+            lines: 70,
+            statements: 70,
+            branches: 75,
+            functions: 65,
```

Comment:
Should these be the same values as the integration.js ones below?

Reply:
These are just the minimum numbers that currently exist. The two different test groups don't have the same coverage levels.